client/my-sites/stats/stats-upsell/style.scss

```
@@ -3,12 +3,11 @@
 @import "@wordpress/base-styles/mixins";
 
 .stats-upsell {
-	max-width: 375px;
 	margin: auto;
 	background: var(--studio-gray-0);
-	@include break-medium {
-		max-width: 700px;
-	}
+	border: 1px solid var(--studio-gray-5);
+	border-radius: 4px;
+
```

```suggestion
    font-family: $font-sf-pro-text;
```
https://github.com/Automattic/wp-calypso/blob/e51efcdb65cdf44bb4e8a5d92ee2c513fd23cf34/packages/components/src/styles/_typography.scss#L5